openapi_python_client/__init__.py

```diff
@@ -18,7 +18,7 @@
 from .utils import snake_case
 
 if sys.version_info.minor < 8:  # version did not exist before 3.8, need to use a backport
-    from importlib_metadata import version
+    from importlib_metadata import version  # pragma: no cover
```

Comment:
```suggestion
    from importlib_metadata import version
```
This is covered by the 3.6 and 3.7 CI tests.